pkg/cmd/provisioning/ibmcloud/delete_service_id.go

```diff
@@ -0,0 +1,81 @@
+package ibmcloud
+
+import (
+	"fmt"
+	"github.com/openshift/cloud-credential-operator/pkg/ibmcloud"
+	"github.com/pkg/errors"
+
+	//"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// NewDeleteServiceIDCmd provides the "delete-service-id" subcommand
+func NewDeleteServiceIDCmd() *cobra.Command {
+	deleteServiceIDCmd := &cobra.Command{
+		Use:              "delete-service-id",
+		Short:            "Delete Service ID",
+		RunE:             deleteServiceIDCmd,
+		PersistentPreRun: initEnvForCreateServiceIDCmd,
+	}
+
+	deleteServiceIDCmd.PersistentFlags().StringVar(&CreateOpts.Name, "name", "", "User-defined name for all created IBM Cloud resources (can be separate from the cluster's infra-id)")
+	deleteServiceIDCmd.MarkPersistentFlagRequired("name")
+	deleteServiceIDCmd.PersistentFlags().StringVar(&CreateOpts.CredRequestDir, "credentials-requests-dir", "", "Directory containing files of CredentialsRequests to delete IAM Roles for (can be created by running 'oc adm release extract --credentials-requests --cloud=ibmcloud' against an OpenShift release image)")
```

Comment:
Up until now, our `delete` subcommands have not required providing the list of CredentialsRequests. I'm not fundamentally opposed to this, but I'd like to hear from the rest of the team (cc @akhil-rane ).

Reply:
As we cannot tag the service ids, we need to construct fullname to delete the service ids. This looks like a better approach to me.